Bottom: cb25f1080902107168aceb40c37309493125577d
Top:    2e5ed82c06743dbc09d48578c5c87648a8352e6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 01:31:06 +0000

hid/gtk/gtkhid-gl.c: Initialise flags variable to of polygon in GhidDrawMask()

(Caught by clang)


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4afaac7..67e610d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1574,6 +1574,7 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.NoHolesValid = 0;
   if (screen)
     polygon.BoundingBox = *screen;
+  polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
   common_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_Free (&polygon.Clipped);
